### PR TITLE
ci: add arm64 and s390x support

### PR DESCRIPTION
**Pull request by yangchiu**
ci: add arm64 and s390x support

For https://github.com/rancher/support-bundle-kit/issues/53

Signed-off-by: Yang Chiu <yang.chiu@suse.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -59,3 +59,185 @@ volumes:
...
+        from_secret: docker_password
+      repo: "rancher/support-bundle-kit"
+      tag: "${DRONE_TAG}-arm64"
+      username:
+        from_secret: docker_username
+    when:
+      instance:
+        - drone-publish.rancher.io
+      ref:
+        - refs/head/master
```
Comment (innobead) [inline .drone.yml:114]: Why need master branch? Please check other arches as well.

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by innobead** [comment]
Need to fix the build script to support arm64 and s390 first before merging.

cc @c3y1huang

File: .drone.yml (hunk 1 above)
Comment (yangchiu) [reply .drone.yml:114]: This part is directly copied from amd64. Shall we remove this from all arches?

Comment (innobead) [reply .drone.yml:114]: If it's not right, then we should remove it for sure.

**Review by innobead — APPROVED** [review]
LGTM

@bk201 Could you help with the review and merge? Thanks. After merging, need to tag a new release for this new change.

**Review by bk201 — APPROVED** [review]
LGTM, thanks.

**Comment by bk201** [comment]
@yangchiu Got some trouble when publishing the s390x port:
https://drone-publish.rancher.io/rancher/support-bundle-kit/94/3/3

**Comment by yangchiu** [comment]
> @yangchiu Got some trouble when publishing the s390x port: https://drone-publish.rancher.io/rancher/support-bundle-kit/94/3/3

probably need this PR https://github.com/rancher/support-bundle-kit/pull/55

**Comment by yangchiu** [comment]
Works now. Images for different platforms can be found on dockerhub: https://hub.docker.com/r/rancher/support-bundle-kit/tags

**Pull request merged**